lib/paper_trail/frameworks/cucumber.rb

```diff
@@ -3,9 +3,9 @@
 # before hook for Cucumber
 Before do
   PaperTrail.enabled = false
-  PaperTrail.enabled_for_controller = true
-  PaperTrail.whodunnit = nil
-  PaperTrail.controller_info = {} if defined? Rails
+  PaperTrail.request.enabled_for_controller = true
+  PaperTrail.request.whodunnit = nil
+  PaperTrail.request.controller_info = {} if defined? Rails
```

Comment:
Should this be `if defined?(::Rails)` to be consistent with the RSpec helper? I don't think there is a Rails module in this library yet but there could be one day

Reply:
Sure, I'm always happy to use absolute constant paths, it's one less thing to worry about.